aws/dynamodb.go

```diff
@@ -0,0 +1,71 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+	"time"
+)
+
+func getAllDynamoTables(session *session.Session, excludeAfter time.Time) ([]*string, error) {
```

Comment:
I just realized this method is not filtering the tables based on the `excludeAfter` param. That is a must-have.